core/App/utils/helpers.ts

```diff
@@ -28,6 +30,51 @@ export function hashToRGBA(i: number) {
   return '#' + '00000'.substring(0, 6 - colour.length) + colour
 }
 
+export function formatTime(time: Date, translate: TFunction, params?: { long?: boolean; format?: string }): string {
```

Comment:
Would creating a type for params help make sure you get the right combination?

```
export type FormatTimeParams = {long: boolean,  format?: undefined}  |
                    {long?: undefined,  format: string}
```

Reply:
neither of the params are required. The format string isn't required because by default we grab it from the translations, and the `long` param isn't required because it will default to the short month 3 letter code